client/src/components/MessageList/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styles from './Style.module.css';
+
+const data = [
+  {
+    self: true,
+    body: 'Hey Alice'
+  },
+  {
+    self: false,
+    body: 'Hi Bob'
+  }
+];
+
+var check1 = 0,
+  check2 = 0;
+
+const MessageList = (props) => {
+  return (
+    <div>
+      {data.map((data, index) => (
+        <div>
```

Comment:
Is this `div` required?

Reply:
Replaced it with fragments.

Reply:
div at line 17 makes sense.
I am talking about line 19 - looks like redundant.

Reply:
Oh okay. I forgot to remove that.